src/Microsoft.AspNetCore.Blazor.Browser.JS/package-lock.json

```diff
@@ -1540,12 +1540,14 @@
         "balanced-match": {
           "version": "1.0.0",
           "bundled": true,
-          "dev": true
+          "dev": true,
+          "optional": true
```

Comment:
@SteveSandersonMS  - is this expected?

Reply:
It's hard to say. NPM has never yet managed to get `package-lock.json` updates to be consistent across machines or NPM versions. It feels like it's getting more consistent over time, but they do keep changing stuff.

On the appearance of `optional:true`, it's probably related to one of these two:

 * https://github.com/npm/npm/issues/19016 (a feature request implemented in NPM 5.6.0)
 * https://npm.community/t/package-lock-json-keeps-changing-between-platforms-and-runs/1129/8 (a bug report that is still open, specifically mentioning the `optional` flag)

I'm not worried about the appearance of this flag because it's supposedly back-compatible. Fine for this PR to include it here, in my view.